wheel/generate_type_stubs.py

```diff
@@ -305,6 +305,19 @@ def confirm_not_included_already_hashed(
     proof: bytes,
 ) -> bool: ...
 
+def get_name_puzzle_conditions(
```

Comment:
did this get reverted? this looks like the wrong name

Reply:
Fixed